fcrepo-http-api/src/main/java/org/fcrepo/http/api/FedoraLdp.java

```diff
@@ -192,21 +193,44 @@ public Response options() {
      * Retrieve the node profile
      *
      * @param rangeValue the range value
-     * @return triples for the specified node
+     * @return the binary or the triples for the specified node
      * @throws IOException if IO exception occurred
      */
     @GET
-    @Produces({TURTLE + ";qs=10", JSON_LD + ";qs=8",
-            N3, N3_ALT2, RDF_XML, NTRIPLES, APPLICATION_XML, TEXT_PLAIN, TURTLE_X,
-            TEXT_HTML, APPLICATION_XHTML_XML, "*/*"})
```

Comment:
Hmm, reflecting on @acoburn's [comment](https://jira.duraspace.org/browse/FCREPO-1840?focusedCommentId=47464&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-47464), I would like to see us move more towards increasing the use of the jaxrs machinery instead of further away.